pkg/gvk/compare_test.go

```diff
@@ -0,0 +1,216 @@
+package gvk
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+	apiextv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic/fake"
+)
+
+var (
+	sourceGroups = []metav1.APIGroup{
```

Comment:
@Danil-Grigorev big 👍 for adding tests!

Reply:
Removing those, as unnecessary addition. The data set for the tests is to huge.